[INFO -- 2017-03-03 07:32:35]
Starting flip_value script
version: 2.0
[INFO -- 2017-03-03 07:32:35]
Initialization strings:
[INFO -- 2017-03-03 07:32:36]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-03-03 07:32:36]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-03 07:32:36]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-03 07:32:36]
(4) file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-03 07:32:36]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-03 07:32:36]
(6) set args 228 2048 16 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_a_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_b_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/gold_2048_m-order_228_ths_16_blocks 1
[INFO -- 2017-03-03 07:32:36]
initSignal:28
[INFO -- 2017-03-03 07:32:36]
endSignal:28
[INFO -- 2017-03-03 07:32:36]
seqSignal:15
[INFO -- 2017-03-03 07:32:36]
faultMode:3
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[EXCEPTION -- 2017-03-03 07:33:08]
pythonException: invalid frame
[INFO -- 2017-03-03 07:33:08]
Backtrace BEGIN:
#0  0x00000000004036da in L_dgemm_185__par_region0_2_0 () at dgemm.c:220
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffe550 in ?? ()
#3  0x0000000000000000 in ?? ()

215	                        for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
216	                            CC_arr(i,j) = 0.0;
217	
218	                    for (kg=kk,k=0; kg<MIN(kk+block,order); k++,kg++)
219	                        for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
220	                            for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
221	                                CC_arr(i,j) += AA_arr(i,k)*BB_arr(j,k);
222	
223	                    for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
224	                        for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)

Backtrace END
Fliping a bit of the value pointed by a pointer
Memory content before bitflip:0x120xe90xe00xe00xdd0x6c0x500x43
Fault Mode: Zero bit-flip
base address to flip value: 0x618ac0
address max offset: 8
Memory content after  bitflip:0x000x000x000x000x000x000x000x00
frame name: L_dgemm_185__par_region0_2_0
symbol name: AA
symbol filename: dgemm.c
symbol line: 187
value: 0x618ac0
value address: 0x7fffffffdf68
Type: The type is a pointer.
Type sizeof: 8

[INFO -- 2017-03-03 07:33:08]
Fault Injection Successful after 32.7818739414s
[INFO -- 2017-03-03 07:33:19]
event type: exit
[INFO -- 2017-03-03 07:33:19]
exit code: 0
